### PR TITLE
[PLAYER-5008] render skin on AD start or end

### PR DESCRIPTION
**Pull request by ooyala-amohnatkin**

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -1464,6 +1464,7 @@ module.exports = function(OO, _, $, W) {
         this.state.adEndTime = this.state.adStartTime + this.state.adVideoDuration;
         this.skin.state.currentPlayhead = 0;
         this.removeBlur();
+        this.renderSkin();
```
Comment (KseniiaSokolovaEpam) [inline js/controller.js]: Is it not automatically rerendered when changing states?

Comment (ooyala-amohnatkin) [reply js/controller.js]: nope
`state` here is just mutable class field

Comment (zelibobla) [reply js/controller.js]: @KseniiaSokolovaEpam setState method doing what you expect. But it's not used here :)

Comment (KseniiaSokolovaEpam) [reply js/controller.js]: @ooyala-amohnatkin @zelibobla ok, thank you, guys!

**Pull request merged**